ios/keyman/Keyman/Keyman/MainViewController.swift

```diff
@@ -932,13 +932,16 @@ class MainViewController: UIViewController, TextViewDelegate, UIActionSheetDeleg
   }
 
   private func profileName(withFont font: Font) -> String? {
-    return font.source.first { !($0.contains(".mobileconfig")) }
+    return font.source.first { $0.hasSuffix(".mobileconfig") }
```

Comment:
we should probably use `$0.lowercased().hasSuffix(".mobileconfig")`. 
we may even want to set `".mobileconfig"` as a static constant instead of using the string